egs/hub4_spanish/s5/local/prepare_unicode_dict.py

```diff
@@ -3,11 +3,19 @@
 # Copyright 2016 Johns Hopkins University (Author: Matthew Wiesner)
 # Apache 2.0
 
-# ======= Prepare data/local directory for babel data with unicode tags ======
-# This script creates all files in the data/local directory needed to generate
-# the kaldi-lang directory. It does not generate the lexicon itself -- for that
-# see the script lexicon/make_unicode_lexicon.py. It expects 
+# ======= Prepare dictionary directory (e.g. data/local) from lexicon.txt =====
+# This script creates all files in the dictionary directory.
 #
+# This script basically takes the place of the prepare_lexicon.pl script for
```

Comment:
I deleted this intentionally because no-one cares how is it done in babel.
I'd say put the original text back and tweak it to taste (list the files to be created if you want). Keep the reference to lexicon/make_unicode_lexicon.py